packages/serverless-cdk-plugin/src/types/cdkPluginConfig.ts

```diff
@@ -1,5 +1,5 @@
-import { Construct } from 'constructs';
+import { ServerlessConstruct } from './serverlessConstruct';
 
 export type CdkPluginConfig = {
```

Comment:
open question: shouldn't we rename this to `ServerlessCdkPluginConfig` to be more in line with the plugin name ? Not sure what's the convention here @CorentinDoue